### ISSUE TITLE
GAM installation hits an error when authorizing scopes

### ISSUE THREAD

**Issue by niklibrato** [jay0lee/GAM#479]
The issue tracker is for reporting product deficiencies. "How do I?" questions should be posted to the discussion forum at https://groups.google.com/group/google-apps-manager. When in doubt, start at the discussion forum and return here only when instructed to do so.

Please confirm the following:
* I have upgraded to the latest GAM release from https://git.io/gamreleases and I still have this issue.
* I am typing the command as described in the GAM Wiki at https://github.com/jay0lee/gam/wiki

Full steps to reproduce the issue:
1. I am following the instructions to the letter, then get to: "Are you ready to authorize GAM to perform G Suite management operations as your admin account? (yes or no) yes"
2. Select the authorized scopes by entering a number... I select 'c)  Continue to authorization'
3. A browser window opens and I get an error:
401. That’s an error.

Error: invalid_client

The OAuth client was not found.

Request Details
scope=email https://www.googleapis.com/auth/admin.directory.group https://www.googleapis.com/auth/admin.directory.orgunit https://www.googleapis.com/auth/admin.directory.user https://www.googleapis.com/auth/admin.directory.device.chromeos https://www.googleapis.com/auth/admin.directory.device.mobile https://apps-apis.google.com/a/feeds/emailsettings/2.0/ https://www.googleapis.com/auth/admin.directory.resource.calendar https://www.googleapis.com/auth/apps.groups.settings https://www.googleapis.com/auth/admin.reports.audit.readonly https://www.googleapis.com/auth/admin.reports.usage.readonly https://www.googleapis.com/auth/apps.licensing https://www.googleapis.com/auth/admin.directory.user.security https://www.googleapis.com/auth/admin.directory.notifications https://www.googleapis.com/auth/siteverification https://www.googleapis.com/auth/admin.directory.userschema https://www.googleapis.com/auth/classroom.rosters https://www.googleapis.com/auth/classroom.courses https://www.googleapis.com/auth/classroom.profile.emails https://www.googleapis.com/auth/classroom.profile.photos https://www.googleapis.com/auth/classroom.guardianlinks.students https://www.googleapis.com/auth/cloudprint https://www.googleapis.com/auth/admin.datatransfer https://www.googleapis.com/auth/admin.directory.customer https://www.googleapis.com/auth/admin.directory.domain https://www.googleapis.com/auth/admin.directory.rolemanagement
login_hint=nik@librato.com
response_type=code
client_id=c223c9509487f62c23c8bbf1527b44855828f013
redirect_uri=http://localhost:8080/
That’s all we know.

Screen shot: https://cl.ly/3e2w3M1Z060o

Expected outcome (what are you trying to do?): 
Complete the GAM setup

Actual outcome (what errors or bad behavior do you see instead?):
see above

**Comment by jay0lee** [comment]
c223c9509487f62c23c8bbf1527b44855828f013 is not a valid client ID which indicates you entered the wrong value for your client ID and secret. Delete client_secrets.json and oauth2service.json (if there is one) then run "gam create project" again and pay close attention to the client ID you enter, make sure it's what's specified in the admin console.

**Comment by niklibrato** [comment]
I followed your instructions, deleted the files and ran `gam create project` paying particular attention to the client ID. I noticed that the format is very different after creating the new key than what it shows under API manager Credentials. https://cl.ly/3h1f3k1g0w2f

Instead of using the client id in the pop up I copied the values, continued with the process of "Enable G Suite Domain-wide Delegation"... then clicked on Client ID in that window (totally different than the one before) and used that client id and the secret from the previous step. 

Then I ran `gam user info` again and it asked me for my G Suite admin email again.
After entering it it asks me to 'Select the authorized scopes by entering a number.' I enter ' c)  Continue to authorization', the browser opens the page with the error again. The Client ID is totally different to what I entered.

https://cl.ly/3y1L2f2F1C2g
